language/transaction-builder/generator/src/generate.rs

```diff
@@ -33,7 +33,7 @@ enum Language {
 )]
 struct Options {
     /// Path to the directory containing ABI files in BCS encoding.
-    abi_directory: PathBuf,
+    abi_directories: Vec<PathBuf>,
```

Comment:
Why do we need multiple abi directory?

Reply:
This is because we currently have both legacy scripts and the new script functions, and their ABI files are stored in two different locations:
- legacy
  - script_abis
- versions
  - xxx
    - script_abis

Alternatively we could duplicate the legacy script ABIs in each version:
- versions
  - xxx
    - legacy_script_abis
    - script_abis

Maybe the latter is actually the more correct thing to do? Just imagine us discontinuing these legacy scripts in a future version... cc @tzakian thoughts?